<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Roman Kositsyn</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>535</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MPSQT</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::ForceTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Empty</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>20</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenNewModel"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Show</string>
    </property>
    <addaction name="actionShowNavigator"/>
    <addaction name="actionShowOutput"/>
   </widget>
   <widget class="QMenu" name="menuWidget">
    <property name="title">
     <string>Widget</string>
    </property>
    <addaction name="actionWidgetModelInfo"/>
    <addaction name="action3DView"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuWidget"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionShowNavigator">
   <property name="text">
    <string>Navigator panel</string>
   </property>
  </action>
  <action name="actionShowOutput">
   <property name="text">
    <string>Output window</string>
   </property>
  </action>
  <action name="actionWidgetModelInfo">
   <property name="text">
    <string>Model information</string>
   </property>
  </action>
  <action name="actionOpenNewModel">
   <property name="text">
    <string>Open new model</string>
   </property>
   <property name="iconText">
    <string>Open new model  </string>
   </property>
  </action>
  <action name="action3DView">
   <property name="text">
    <string>3D View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
